web/annotation_layer_builder.css

```diff
@@ -93,6 +93,64 @@
   width: 115%;
 }
 
+.annotationLayer .checkboxWidgetAnnotation label,
+.annotationLayer .radioButtonWidgetAnnotation label {
+  background-color: rgb(222, 229, 255);
```

Comment:
Why not try to fix this properly instead, by adding `getOperatorList` overrides to the new class (in `src/core/annotation.js`)?
Similar to the existing `TextWidgetAnnotation`, but perhaps a bit simpler for now:
```javascript
Util.inherit(ButtonWidgetAnnotation, WidgetAnnotation, {
  getOperatorList:
      function ButtonWidgetAnnotation_getOperatorList(evaluator, task,
                                                      renderForms) {
    var operatorList = new OperatorList();

    // Do not render form elements on the canvas when interactive forms are
    // enabled. The display layer is responsible for rendering them instead.
    if (renderForms) {
      return Promise.resolve(operatorList);
    }

    if (this.appearance) {
      return Annotation.prototype.getOperatorList.call(this, evaluator, task,
                                                       renderForms);
    }
    return Promise.resolve(operatorList);
  }
```